db/primary_migrate/20220411224513_add_submit_at_to_doc_auth_logs.rb

```diff
@@ -0,0 +1,12 @@
+class AddSubmitAtToDocAuthLogs < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+
+  def change
+    add_column :doc_auth_logs, :verify_submit_at, :datetime
+    add_column :doc_auth_logs, :verify_phone_submit_count, :integer, default: 0
+    add_column :doc_auth_logs, :verify_phone_submit_at, :datetime
+    add_column :doc_auth_logs, :document_capture_submit_at, :datetime
+
+    add_index :doc_auth_logs, :issuer, algorithm: :concurrently
```

Comment:
We should add the index in its own migration because on larger tables those tend to timeout